testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxBoundedContext.java

```diff
@@ -283,6 +283,60 @@ public T receivesEvents(Message firstEvent, Message secondEvent, Message... othe
         return this.receivesEvents(asList(firstEvent, secondEvent, otherEvents));
     }
 
+    /**
+     * Sends off a provided event to the Bounded Context as event from an external source.
+     *
+     * @param messageOrEvent
+     *         an event message or {@link io.spine.core.Event}. If an instance of {@code Event} is
+     *         passed, it will be posted to {@link io.spine.server.integration.IntegrationBus}
+     *         as is.
+     *         Otherwise, an instance of {@code Event} will be generated basing on the passed
+     *         event message and posted to the bus.
+     * @return current instance
+     * @apiNote Returned value can be ignored when this method invoked for test setup
+     */
+    @CanIgnoreReturnValue
+    public T receivesExternalEvent(Message messageOrEvent) {
```

Comment:
In case of a plain message posted, it's not clear how to set the mandatory `ExternalMessage.bounded_context_name` field.